app/views/content/support-for-early-career-teachers.md

```diff
@@ -6,7 +6,7 @@ description: |-
 related_content:
     Initial teacher training : "/train-to-be-a-teacher/initial-teacher-training"
     Advice from an early career teacher : "/blog/being-myself-in-the-classroom"
-    Managing behaviour in the classroom : "/blog/managing-behaviour-in-the-classroom"
+    Managing behaviour in the classroom : "/is-teaching-right-for-me/behaviour-management"
```

Comment:
This all works fine - but are there any other places around the site that we'd want to add a related content link (or inline link)?

Reply:
I'll have a look through the pages to see if there are any other relevant places we can link to.

Reply:
We can add a related link on this page as it talks about what to expect in schools and how teachers manage a classroom https://getintoteaching.education.gov.uk/is-teaching-right-for-me/get-school-experience

We can add an inline link in the last step of the how to become a teacher page where it talks about behaviour management https://getintoteaching.education.gov.uk/steps-to-become-a-teacher

Inline link in the What you'll learn section bullet point **manage behaviour effectively to ensure a good and safe learning environment**  and we could also have a related link https://getintoteaching.education.gov.uk/train-to-be-a-teacher/initial-teacher-training